editor/inspector_dock.cpp

```diff
@@ -408,29 +408,27 @@ void InspectorDock::update(Object *p_object) {
 
 	current = p_object;
 
-	if (!p_object) {
-		open_docs_button->set_disabled(true);
-		object_menu->set_disabled(true);
-		warning->hide();
-		search->set_editable(false);
-		editor_path->clear_path();
-		return;
-	}
-
-	bool is_resource = p_object->is_class("Resource");
-	bool is_node = p_object->is_class("Node");
-
-	object_menu->set_disabled(false);
-	search->set_editable(true);
-	editor_path->enable_path();
+	const bool is_object = p_object;
```

Comment:
This works but I'd prefer to make the cast obvious here so that it doesn't look like a wrong assignment.
```suggestion
	const bool is_object = p_object != nullptr;
```